serverless/cmd/sequence/main.go

```diff
@@ -34,12 +36,13 @@ import (
 var (
 	storageDir = flag.String("storage_dir", "", "Root directory to store log data.")
 	entries    = flag.String("entries", "", "File path glob of entries to add to the log.")
-	create     = flag.Bool("create", false, "Set when creating a new log to initialise the structure.")
 )
 
 func main() {
 	flag.Parse()
 
+	pubKey := os.Getenv("SERVERLESS_LOG_PUBLIC_KEY")
```

Comment:
same `len` and `err` checks in this file too.

Reply:
Done